src/index.ts

```diff
@@ -10,6 +12,26 @@ export {
    Router,
 };
 
+/**
+ * Creates an async Lambda handler function from an Application instance.
+ *
+ * @param app The Application instance to use for handling requests
+ * @returns An async Lambda handler function
+ *
+ * @example
+ * ```typescript
+ * const app = new Application();
+ * app.get('/hello', (req, res) => res.send('Hello World!'));
+ *
+ * export const handler = createAsyncHandler(app);
+ * ```
+ */
+export function createAsyncHandler(app: Application): (event: RequestEvent, context: Context) => Promise<ResponseResult> {
+   return async (event: RequestEvent, context: Context): Promise<ResponseResult> => {
```

Comment:
The `async` keyword is unnecessary since `runAsync()` already returns a Promise.

```suggestion
   return (event: RequestEvent, context: Context): Promise<ResponseResult> => {
```